scalding-thrift-macros-fixtures/src/main/scala/com/twitter/scalding/thrift/macros/scalathrift/TestEnum.scala

```diff
@@ -0,0 +1,94 @@
+/**
+ * Generated by Scrooge
+ *   version: ?
+ *   rev: ?
+ *   built at: ?
+ */
+package com.twitter.scalding.thrift.macros.scalathrift
```

Comment:
@ianoc Do you still have the original thrift file? We will have to generate these on the fly if we dont want to check them in.